integrations/kube-agent-updater/pkg/controller/constants.go

```diff
@@ -24,10 +24,11 @@ import (
 
 const (
 	// Teleport container name in the `teleport-kube-agent` Helm chart
-	teleportContainerName = "teleport"
-	defaultRequeue        = 30 * time.Minute
-	reconciliationTimeout = 2 * time.Minute
-	kubeClientTimeout     = 1 * time.Minute
+	teleportContainerName        = "teleport"
+	defaultRequeue               = 30 * time.Minute
+	reconciliationTimeout        = 2 * time.Minute
+	kubeClientTimeout            = 1 * time.Minute
+	skipReconciliationAnnotation = "kubernetes.teleport.dev/skip-reconciliation"
```

Comment:
I assume this values comes from the cloud repo, correct?
Can we add a comment or a link?

Reply:
Hmmm, I did not check we had an annotation in the cloud repo and just came up with the name. I'll check 😅